client/src/components/Header.js

```diff
@@ -56,6 +57,35 @@ function Header() {
     setAnchorEl(null);
   };
 
+  const logout = () => {
+    let status
+    fetch("/logout", {
+      method: "POST",
+      credentials: "include"
+    })
+    .then(res => {
+        status = res.status
+
+        if (status < 500) return res.json();
+        else throw Error("Server error");
+    })
+    .then(res=> {
+        if (status === 200) {
+          history.push("/login")
+          alert(res.response)
+        }
+        else if (status === 500) {
+          alert(res.response)
+        }
+        setAnchorEl(null)
+      })
```

Comment:
I think there is still possibility of returning 40x status code (ex. BE server down or connection fail), and we might need to add and `else` at bottom to` throw Error` to handle it

Reply:
okk added else after else if to throw error